src/v2/view-builder/views/phone/EnrollAuthenticatorDataPhoneView.js

```diff
@@ -0,0 +1,173 @@
+import { _, loc, Model } from 'okta';
```

Comment:
This is just moved from `EnrollAuthenticatorPhoneView`, no changes in this file.